packages/fxa-settings/src/pages/SigninTokenCode/en.ftl

```diff
@@ -7,10 +7,10 @@ signin-token-code-heading = Enter confirmation code<span> for your { -product-fi
 # { $email } represents the email that the user entered to sign in
 signin-token-code-instruction = Enter the code that was sent to { $email } within 5 minutes.
 signin-token-code-input =
-  .label = Confirmation code
-  .placeholder = Enter 6-digit code
+  .label = Enter 6-digit code
```

Comment:
Even when changing or adding an attribute to an existing string, you'll need to make a new ID for the string.

Reply:
Meant to change that. Sorry! I'll update the ID to `signin-token-code-input-label`

Reply:
Double checking just in case but the .label attribute has been deleted.

Reply:
Thanks for double checking, spotted an error in this update - latest push has fixed ids.

Reply:
See the new IDs now, looks good 👍